server/routes/incidents.js

```diff
@@ -17,26 +19,47 @@ module.exports = function CreateReportRoutes({ reportService, offenderService })
 
   return {
     redirectToHomePage: async (req, res) => {
-      res.redirect('/my-statements')
+      res.redirect('/your-statements')
     },
 
     viewReportSent: async (req, res) => {
       res.render('pages/report-sent', { data: res.locals.formObject, reportId: req.params.reportId, links })
     },
 
-    viewMyReports: async (req, res) => {
+    viewYourReports: async (req, res) => {
       const awaiting = await reportService.getReports(req.user.username, ReportStatus.IN_PROGRESS)
       const completed = await reportService.getReports(req.user.username, ReportStatus.SUBMITTED)
 
       const namesByOffenderNumber = await getOffenderNames(res.locals.user.token, [...awaiting, ...completed])
       const awaitingReports = awaiting.map(toReport(namesByOffenderNumber))
       const completedReports = completed.map(toReport(namesByOffenderNumber))
 
-      res.render('pages/my-reports', {
+      res.render('pages/your-reports', {
         awaitingReports,
         completedReports,
-        selectedTab: 'my-reports',
+        selectedTab: 'your-reports',
       })
     },
+
+    viewYourReport: async (req, res) => {
+      const { reportId } = req.params
+
+      const { id, form = {}, incidentDate, bookingId } = await reportService.getReport(req.user.username, reportId)
```

Comment:
Might be worth checking user input here and return a bad request etc.

Reply:
Is there a general catch-all error in place that returns a standard error template?

Reply:
👍 We do need to check this. We don't have an error page yet - we have a story for it.